pool/web/view/page/page_admin_custom_fields.ml

```diff
@@ -553,34 +554,36 @@ let field_form
               ~orientation:`Horizontal
               ~value:
                 (value (fun f ->
-                   (f |> admin).Admin.hint
-                   |> CCOption.map_or ~default:"" Admin.Hint.value))
+                   f
+                   |> admin_hint
+                   |> CCOption.map_or ~default:"" AdminHint.value))
               ~flash_fetcher
           ; checkbox_element Message.Field.Overwrite (fun f ->
-              (f |> admin).Admin.overwrite |> Admin.Overwrite.value)
+              f |> admin_overwrite |> AdminOverwrite.value)
           ; checkbox_element
               ~disabled:
                 (custom_field
-                |> CCOption.map_or ~default:false (fun f ->
-                     (f |> admin).Admin.view_only |> Admin.ViewOnly.value))
+                |> CCOption.map_or
+                     ~default:false
+                     (admin_view_only %> AdminViewOnly.value))
               ~help:Pool_common.I18n.CustomFieldAdminInputOnly
               Message.Field.AdminInputOnly
-              (fun f -> (f |> admin).Admin.input_only |> Admin.InputOnly.value)
+              (admin_input_only %> AdminInputOnly.value)
           ; checkbox_element
               ~help:Pool_common.I18n.CustomFieldAdminViewOnly
               Message.Field.AdminViewOnly
-              (fun f -> (f |> admin).Admin.view_only |> Admin.ViewOnly.value)
+              (admin_view_only %> AdminViewOnly.value)
           ]
       ; div
           ~a:[ a_class [ "stack" ] ]
           [ checkbox_element
               ~disabled:
                 (custom_field
                 |> CCOption.map_or ~default:false (fun f ->
-                     (f |> admin).Admin.input_only |> Admin.InputOnly.value
+                     f |> admin_input_only |> AdminInputOnly.value
```

Comment:
```suggestion
                     (admin_input_only %> AdminInputOnly.value
```

Reply:
I don't think that works, `f` is used on both sides of `||`

Reply:
You're right, didn't saw the short circuit `||`.

Please update the other `fun f ->` in this file code line 231, (562 - already mentioned), 587.